diff --git a/deps/libwebrtc/src/include/pc/srtp_transport.h b/deps/libwebrtc/src/include/pc/srtp_transport.h
--- a/deps/libwebrtc/src/include/pc/srtp_transport.h
+++ b/deps/libwebrtc/src/include/pc/srtp_transport.h	(date 1728384908960)
@@ -113,6 +113,10 @@
   // If the writable state changed, fire the SignalWritableState.
   void MaybeUpdateWritableState();

+  // Decrypts/verifies an invidiual RTP/RTCP packet.
+  // If an HMAC is used, this will decrease the packet size.
+  bool UnprotectRtp(void* data, int in_len, int* out_len);
+
  private:
   void ConnectToRtpTransport();
   void CreateSrtpSessions();
@@ -135,10 +139,6 @@
                   int64_t* index);
   bool ProtectRtcp(void* data, int in_len, int max_len, int* out_len);

-  // Decrypts/verifies an invidiual RTP/RTCP packet.
-  // If an HMAC is used, this will decrease the packet size.
-  bool UnprotectRtp(void* data, int in_len, int* out_len);
-
   bool UnprotectRtcp(void* data, int in_len, int* out_len);
 
   bool MaybeSetKeyParams();
